kafka_utils/kafka_cluster_manager/cluster_info/genetic_balancer.py

```diff
@@ -564,6 +566,8 @@ def _score(self, state, score_movement=True):
                 (1 - state.broker_leader_weight_cv / sqrt(len(state.brokers)))
             score += self.args.topic_broker_imbalance_score_weight * \
                 (1 - state.weighted_topic_broker_imbalance)
+            score += self.args.broker_partition_count_score_weight * \
+                (1 - state.broker_partition_count_cv)
```

Comment:
Good catch. I fixed it. I also adjusted the default weights to keep the score between 0 and 1.